lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,15 @@
+namespace :broadcasts do
+  desc "Send Welcome Notifications once a day"
+  task send_welcome_notification_flow: :environment do
+    # In order to prevent new users from receiving multiple welcome notifications in a day,
+    # a feature_live_date is required. The script will only be effective after feature_live_date
+    # and will ultimately be superseded by 7.days.ago when it's larger than feature_live_date
+    feature_live_date = SiteConfig.welcome_notifications_live_date
+    week_ago = 7.days.ago
+    latest_date = feature_live_date > week_ago ? feature_live_date : week_ago
```

Comment:
very nice way to handle these two scenarios! 🥇

Reply:
```suggestion
    latest_date = notifications_live_at > week_ago ? notifications_live_at : week_ago
```